bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=opentelemetry-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.27.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.28.0
```

Comment:
I assume you did not intend to update operator-sdk? :sweat_smile:

Reply:
my local version upgraded, i can force to use 1.27 though if we are worried about the upgrades.

Reply:
please extract this to a separete PR